lib/kubeclient/common.rb

```diff
@@ -115,9 +115,9 @@ def get_entities(entity_type, klass, options)
           params['params'] = { labelSelector: options[:label_selector] }
         end
 
-        # TODO: namespace support?
+        ns_prefix = build_namespace_prefix(options[:namespace])
```

Comment:
Both `:label_selector` (maybe to be renamed to `:selector` depending on kubernetes terminology) and `namespace` should be additional arguments (as in `get_entity`).